release/models/platform/openconfig-platform.yang

```diff
@@ -65,7 +65,13 @@ module openconfig-platform {
     (presence or absence of a component) and state (physical
     attributes or status).";
 
-  oc-ext:openconfig-version "0.24.0";
+  oc-ext:openconfig-version "0.25.0";
```

Comment:
There are no edits to `openconfig-platform` or its submodule `openconfig-platform-common` and `openconfig-platform-storage` is entirely new and augments into the tree so I'm curious why versions are being touched on any modules other than the new module?

Reply:
Fixed

Reply:
Actually, our CI check catches this as a module revision must be in sync with a submodule (openconfig-platform-common.yang ) needing to be in sync 

Ref:
https://gist.github.com/OpenConfigBot/15bc064de0245e83f65bad75c3870f35

Reply:
I think these 2 modules can be dropped out of this patch set - the master version for both is 0.27.0 and neither are being touched as part of this PR